agent/src/main/java/com/cloud/agent/properties/AgentProperties.java

```diff
@@ -659,6 +659,14 @@ public class AgentProperties{
      */
     public static final Property<Integer> STOP_SCRIPT_TIMEOUT = new Property<>("stop.script.timeout", 120);
 
+    /**
+     * Time (in seconds) to wait for scripts to complete.<br>
+     * This is not used for every script execution.<br>
+     * Data type: Integer.<br>
+     * Default value: <code>60</code>
+     */
+    public static final Property<Integer> AGENT_SCRIPT_TIMEOUT = new Property<>("agent.script.timeout", 60);
```

Comment:
@vishesh92 we need to add this in agent.properties to make it configurable

Reply:
We don't need to. If the user adds it in the file, it will get picked up.

Reply:
yeah correct but that has to be known to the operator right, we normally put them in the properties file and comment them 

eg.

# Instance Conversion from Vmware to KVM through virt-v2v. Enable verbose mode
# virtv2v.verbose.enabled=false